.github/workflows/github.yml

```diff
@@ -205,11 +205,18 @@ jobs:
       env:
         PULL_REQUEST: ${{ github.event.pull_request.number }}
 
-    - name: Configure
+    - name: Configure (ccache)
+      if: ${{ matrix.platform != 'msvc' }}
       run: |
         if [[ ${{matrix.preset}} == linux-gcc-test ]]; then GCC14=1; fi
         cmake -DENABLE_CCACHE:BOOL=ON --preset ${{ matrix.preset }} ${GCC14:+-DCMAKE_C_COMPILER=gcc-14 -DCMAKE_CXX_COMPILER=g++-14}
 
+    - name: Configure (no ccache)
```

Comment:
instead of creating new step you could simply check if `preset` ends with `ccache` and add `-DENABLE_CCACHE:BOOL=ON` based on that